google/resource-snippets/cloudresourcemanager-v2/folders/folder.yaml

```diff
@@ -15,13 +15,11 @@
 imports:
   - path: folder.jinja
```

Comment:
Nit: Let's need the spacing here consistent across blocks and either unindent this line or indent everything one level under `resources` and `outputs`.

Reply:
Thanks for pointing it out. Done.